library/core/src/panic.rs

```diff
@@ -44,27 +70,6 @@ pub macro panic_2015 {
     }),
 }
 
-#[doc(hidden)]
-#[unstable(feature = "edition_panic", issue = "none", reason = "use panic!() instead")]
-#[allow_internal_unstable(panic_internals, const_format_args)]
-#[rustc_diagnostic_item = "core_panic_2021_macro"]
-#[rustc_macro_transparency = "semitransparent"]
-#[cfg(feature = "panic_immediate_abort")]
-pub macro panic_2021 {
-    () => (
-        $crate::panicking::panic("explicit panic")
-    ),
-    // Special-case the single-argument case for const_panic.
-    ("{}", $arg:expr $(,)?) => ({
-        $crate::panicking::panic_display(&$arg);
-    }),
-    ($($t:tt)+) => ({
-        // Semicolon to prevent temporaries inside the formatting machinery from
-        // being considered alive in the caller after the panic_fmt call.
-        $crate::panicking::panic_fmt($crate::const_format_args!($($t)+));
-    }),
-}
```

Comment:
Why has this been removed?

Reply:
With this patch the abort/unwind impls will become identical.

Reply:
I see.

My apologies, the initial comment I had left had been from skimming this deletion, finding it odd, trying to figure out if it had merely been moved into another part of restructuring of the panic surface, and not realizing what actually happened.